src/nystrom.jl

```diff
@@ -156,6 +156,26 @@ end
 _green_multiplier(x::Tuple, Q::Quadrature) = _green_multiplier(SVector(x), Q)
 _green_multiplier(x::QuadratureNode, Q::Quadrature) = _green_multiplier(coords(x), Q)
 
+"""
+    _green_multiplier(s::Symbol)
+
+Return `-1.0` if `s == :inside`, `0.0` if `s == :outside`, and `-0.5` if `s ==
+:on`; otherwise, throw an error. The orientation is relative to the normal of
+the bounding curve/surface.
+"""
```

Comment:
I think it would make sense to document this at a more visible place in the API.

Reply:
The `green_multiplier` keyword argument is now documented in both the `bdim_correction` and `vdim_correction` methods. In both cases, a reference to the method `_green_multiplier` is included. 

Since both `bdim_correction` and `vdim_correction` are *not* part of the API, I think this is enough for now as far as documenting the usage of this argument goes.